CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.66.0
+
+- When configuring Copr chroot (target in Packit terminology) specific configuration, make sure to specify `additional_modules` as a string: module names separated with a comma, example: "httpd:2.4,python:4". (#1826)
```

Comment:
I couldn't parse the colon, this would make it readable for me:
```suggestion
- When configuring Copr chroot (target in Packit terminology) specific configuration, make sure to specify `additional_modules` as a string containing module names separated with a comma, example: "httpd:2.4,python:4". (#1826)
```

Reply:
noted: I should drop using "colon" so much :D